### PR TITLE
CB-14924 : Added boolean option for skipping datalake backup during upgrade process.

### PR DESCRIPTION
**Pull request by sxxgrc**
Jira: https://jira.cloudera.com/browse/CB-14924

Just added a new field in the request which is checked by the flow trigger. I also had to modify the tests, and did some slight cleanup of the classes I worked in.

This has not been tested, I will do that locally soon, but wanted eyes on the change.

### CODE REVIEW

File: datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxUpgradeRequest.java [hunk 1]
```diff
@@ -57,6 +59,18 @@ public void setDryRun(Boolean dryRun) {
         this.dryRun = dryRun;
     }
 
+    public Boolean getSkipBackup() {
```
Comment (yg-gd) [inline datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxUpgradeRequest.java:62]: Method not needed

**Review by yg-gd — APPROVED** [review]

**Comment by sxxgrc** [comment]
Things look good so I'm merging this in.

**Comment by sxxgrc** [comment]
Or uh, actually I can't merge this in. Maybe only @hortonworks/cloudbreak-raven can merge this in?

**Review by bbihari — APPROVED** [review]
LGTM

**Review by kkalvagadda1 — APPROVED** [review]

**Comment by sodre90** [comment]
Should we wait for duck review? @schfeca75

**Review by tiborpopovics — APPROVED** [review]

**Pull request merged**